### ISSUE TITLE
Tcp header length calculation may be wrong

### ISSUE THREAD

**Issue by 0xn0ne** [williamfhe/godivert#3]
When I was using binding. I got this
```
goroutine 20 [running]:
github.com/williamfhe/godivert/header.NewTCPHeader(...)
        C:/Users/Blueshark-JF/go/src/github.com/williamfhe/godivert/header/tcp.go:16
github.com/williamfhe/godivert.(*Packet).ParseHeaders(0xc000099b60)
        C:/Users/Blueshark-JF/go/src/github.com/williamfhe/godivert/packet.go:42 +0x5a2
github.com/williamfhe/godivert.(*Packet).VerifyParsed(0xc000099b60)
        C:/Users/Blueshark-JF/go/src/github.com/williamfhe/godivert/packet.go:187 +0x47
github.com/williamfhe/godivert.(*Packet).DstIP(0xc000099b60, 0x2, 0x2, 0x7)
        C:/Users/Blueshark-JF/go/src/github.com/williamfhe/godivert/packet.go:102 +0x32
main.checkPacket(0xc000052610, 0xc0000980c0)
        C:/Users/Blueshark-JF/go/src/vtap/old2.go:75 +0x25f
created by main.NewProxy
        C:/Users/Blueshark-JF/go/src/vtap/old2.go:319 +0xff
exit status 2
```

I found out when I was following the mistake, in *github.com\williamfhe\godivert\header*

```
func NewTCPHeader(raw []byte) *TCPHeader {
	hdrLen := raw[12] >> 2
	return &TCPHeader{
		Raw: raw[:hdrLen],
	}
}
```

I think maybe the following code will be more accurate

```
(packet.Raw[ipHeaderLenght+12]&0xF>>4) * 5
```

**Comment by williamfhe** [comment]
You were right, the calculation was wrong.
It's been fixed with commit 2ec00c2b6424863ba5eb9811812746bec540cac4

**Comment by 0xn0ne** [comment]
Ooh, finally updated, works well on my code. I like this binding very much. QwQ
